pkg/hash/reader.go

```diff
@@ -52,13 +52,12 @@ func NewReader(src io.Reader, size int64, md5Hex, sha256Hex string) (*Reader, er
 	if err != nil {
 		return nil, err
 	}
+
 	var (
 		md5Hash    hash.Hash
```

Comment:
md5Hash is not required - we can directly use `md5Hash: md5.New()` during Reader init.

Reply:
Sure let me do that.